state/execution.go

```diff
@@ -145,10 +144,9 @@ func (blockExec *BlockExecutor) CreateProposalBlock(
 	}
 
 	// update the block with the response from PrepareProposal
-	block.Data, _ = types.DataFromProto(&cmtproto.Data{
-		Txs:  rpp.Txs[:len(rpp.Txs)-1],
-		Hash: rpp.Txs[len(rpp.Txs)-1],
```

Comment:
The `Hash` is left unset, is it intentional?

Reply:
Yes. We could probably remove it from the `Data` struct if we wanted to

Reply:
So, wouldn't that be a breaking behaviour, namely it has been previously set and with the new change it is not?

Reply:
Actually going to revert this because I don't think it makes sense to break app just yet